bin/instant-workstation

```diff
@@ -137,7 +147,39 @@ then
 fi
 if test -n "$sysctl"
 then
-	command="$command ; /usr/sbin/sysrc -f /etc/sysctl.conf $sysctl";
+	command="$command ; /sbin/sysctl -f /etc/sysctl.conf $sysctl";
 fi
 
+echo $command
 su root -c "$command"
+
+### VMWARE Post installation
+#
+#
+if test 0405 = "${vga_vendor}"
```

Comment:
Maybe add `is_vbox` and `is_vmware` somewhere, with suitable tests to set them, so that this becomes more readable?